src/gebt_poc/model.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "src/gebt_poc/section.h"
+
+namespace openturbine::gebt_poc {
+
+/// Class to manages different aspects of the beam finite element model
+class Model {
+public:
+    Model(std::string name) : name_(name) {}
+
+    /// Returns the name of the model
+    inline std::string GetName() const { return name_; }
+
+    /// Adds a section to the model
+    void AddSection(Section section) { sections_.emplace_back(section); }
+
+    /// Returns the sections of the model
+    inline const std::vector<Section>& GetSections() const { return sections_; }
+
+private:
+    std::string name_;               //< Name of the model
+    std::vector<Section> sections_;  //< Sections of the beam
```

Comment:
Should our Sections be stored in a vector?  std::vector can't be used on GPUs, so we'll need to somehow make this data accessible there.

Reply:
I converted the above into a `Kokkos::View<Section*>`, along with other necessary changes required by it.